### DISCUSSION TITLE
Project Submission button not getting enabled in FRT

### DISCUSSION THREAD

**Discussion by Nishku2909** [github/india#882] [Students]
Even after adding Microsoft Azure skill on LinkedIn ..project submission button is not getting enabled...i have been waiting it to get activated since a week....i am unable to submit my project...what's the way out....please convey if somebody knows....

**Comment by Diksha4111** [comment]
> Even after adding Microsoft Azure skill on LinkedIn ..project submission button is not getting enabled...i have been waiting it to get activated since a week....i am unable to submit my project...what's the way out....please convey if somebody knows....

Same problem.....if anyone knows what to do please help
